content/zh-cn/docs/reference/access-authn-authz/node.md

```diff
@@ -13,193 +12,215 @@ reviewers:
 
-### 相对于以前使用 RBAC 的版本的更新
 <!--
 ### Upgrades from previous versions using RBAC
 -->
+### 相对于以前使用 RBAC 的版本的更新   {#upgrades-from-previous-versions-using-rbac}
 
-升级的 1.7 之前的使用 [RBAC](/zh/docs/reference/access-authn-authz/rbac/) 的集群将继续按原样运行，因为 `system:nodes` 组绑定已经存在。
 <!--
 Upgraded pre-1.7 clusters using [RBAC](/docs/reference/access-authn-authz/rbac/) will continue functioning as-is because the `system:nodes` group binding will already exist.
 -->
+升级的 1.7 之前的使用 [RBAC](/zh-cn/docs/reference/access-authn-authz/rbac/)
+的集群将继续按原样运行，因为 `system:nodes` 组绑定已经存在。
 
-如果集群管理员希望开始使用 `Node` 鉴权器和 `NodeRestriction` 准入插件来限制节点对 API 的访问，这一需求可以通过下列操作来完成且不会影响已部署的应用：
 <!--
 If a cluster admin wishes to start using the `Node` authorizer and `NodeRestriction` admission plugin
 to limit node access to the API, that can be done non-disruptively:
- -->
+-->
+如果集群管理员希望开始使用 `Node` 鉴权器和 `NodeRestriction` 准入插件来限制节点对
+API 的访问，这一需求可以通过下列操作来完成且不会影响已部署的应用：
 
-1. 启用 `Node` 鉴权模式 (`--authorization-mode=Node,RBAC`) 和 `NodeRestriction` 准入插件
-2. 确保所有 kubelet 的凭据符合组/用户名要求
-3. 审核 apiserver 日志以确保 `Node` 鉴权器不会拒绝来自 kubelet 的请求（日志中没有持续的 `NODE DENY` 消息）
-4. 删除 `system:node` 集群角色绑定
 <!--
 1. Enable the `Node` authorization mode (`--authorization-mode=Node,RBAC`) and the `NodeRestriction` admission plugin
 2. Ensure all kubelets' credentials conform to the group/username requirements
 3. Audit apiserver logs to ensure the `Node` authorizer is not rejecting requests from kubelets (no persistent `NODE DENY` messages logged)
 4. Delete the `system:node` cluster role binding
 -->
+1. 启用 `Node` 鉴权模式 (`--authorization-mode=Node,RBAC`) 和 `NodeRestriction` 准入插件
+2. 确保所有 kubelet 的凭据符合组/用户名要求
+3. 审核 API 服务器日志以确保 `Node` 鉴权器不会拒绝来自 kubelet 的请求（日志中没有持续的 `NODE DENY` 消息）
+4. 删除 `system:node` 集群角色绑定
 
-### RBAC 节点权限
 <!--
 ### RBAC Node Permissions
 -->
+### RBAC 节点权限   {#rbac-node-permissions}
 
-在 1.6 版本中，当使用 [RBAC 鉴权模式](/zh/docs/reference/access-authn-authz/rbac/) 时，`system:nodes` 集群角色会被自动绑定到 `system:node` 组。
 <!--
 In 1.6, the `system:node` cluster role was automatically bound to the `system:nodes` group when using the [RBAC Authorization mode](/docs/reference/access-authn-authz/rbac/).
 -->
+在 1.6 版本中，当使用 [RBAC 鉴权模式](/zh-cn/docs/reference/access-authn-authz/rbac/)
+时，`system:nodes` 集群角色会被自动绑定到 `system:node` 组。
 
-在 1.7 版本中，不再推荐将 `system:nodes` 组自动绑定到 `system:node` 角色，因为节点鉴权器通过对 secret 和 configmap 访问的额外限制完成了相同的任务。
-如果同时启用了 `Node` 和 `RBAC` 授权模式，1.7 版本则不会创建 `system:nodes` 组到 `system:node` 角色的自动绑定。
 <!--
 In 1.7, the automatic binding of the `system:nodes` group to the `system:node` role is deprecated
 because the node authorizer accomplishes the same purpose with the benefit of additional restrictions
 on secret and configmap access. If the `Node` and `RBAC` authorization modes are both enabled,
 the automatic binding of the `system:nodes` group to the `system:node` role is not created in 1.7.
 -->
+在 1.7 版本中，不再推荐将 `system:nodes` 组自动绑定到 `system:node`
+角色，因为节点鉴权器通过对 secret 和 configmap 访问的额外限制完成了相同的任务。
```

Comment:
```suggestion
角色，因为节点鉴权器通过对 Secret 和 ConfigMap 访问的额外限制完成了相同的任务。
```
I'm not sure if I'm right about this, just for your reference

Reply:
It make this document more readable, thanks!